webapp/client/public/css/base.css

```diff
@@ -2,580 +2,608 @@
 
 .alert-success {
-    background-color: var(--success-color);
-    color: white;
+  background-color: var(--success-color);
+  color: white;
 }
 
 .alert-warning {
-    background-color: var(--error-color);
-    color: white;
+  background-color: var(--error-color);
+  color: white;
 }
 
 .form-row {
-    margin-left: 0;
+  margin-left: 0;
 }
 
 .steps-number {
-    display: flex;
-    width: 32px;
-    height: 32px;
-    border-radius: 16px;
-    align-items: center;
-    justify-content: center;
-    background-color: #bfd4ed;
+  display: flex;
+  width: 32px;
+  height: 32px;
+  border-radius: 16px;
+  align-items: center;
+  justify-content: center;
+  background-color: #bfd4ed;
 }
 
 .step-active small {
-    color: #007bff;
-    font-weight: bold;
+  color: #007bff;
+  font-weight: bold;
 }
 
 .step-active .steps-number {
-    background-color: #007bff;
-    color: var(--inverse-text-color);
-    font-weight: bold;
+  background-color: #007bff;
+  color: var(--inverse-text-color);
+  font-weight: bold;
 }
 
 .cursor-pointer {
-    cursor: pointer;
+  cursor: pointer;
 }
 
 #skip-to-content-div {
-    background-color: var(--bg-highlight-color);
-    display: flex;
-    width: 100%;
+  background-color: var(--bg-highlight-color);
+  display: flex;
+  width: 100%;
 }
 
 #skip-to-content-link {
-    position: absolute;
-    left: -10000px;
-    top: auto;
-    width: 1px;
-    height: 1px;
-    overflow: hidden;
+  position: absolute;
+  left: -10000px;
+  top: auto;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
 }
 
 #skip-to-content-link:focus {
-    position: static;
-    height: auto;
-    width: auto;
-    margin-top: var(--spacing-02);
-    margin-bottom: var(--spacing-01);
-    border: none;
-    font-size: var(--text-base);
-    font-family: var(--font-bold);
-    color: var(--text-color);
+  position: static;
+  height: auto;
+  width: auto;
+  margin-top: var(--spacing-02);
+  margin-bottom: var(--spacing-01);
+  border: none;
+  font-size: var(--text-base);
+  font-family: var(--font-bold);
+  color: var(--text-color);
 }
 
 @media (max-width: 1024px) {
-    #skip-to-content-link:focus {
-        margin-top: var(--spacing-04);
-        margin-bottom: var(--spacing-03);
-    }
+  #skip-to-content-link:focus {
+    margin-top: var(--spacing-04);
+    margin-bottom: var(--spacing-03);
+  }
 }
 
 .hidden {
-    display: none;
+  display: none;
 }
 
 .section-title {
-    margin-bottom: var(--spacing-05);
-    margin-top: var(--spacing-11);
+  margin-bottom: var(--spacing-05);
+  margin-top: var(--spacing-11);
+}
+
+.steps_info_box_with_icon {
+  padding: 32px 64px 48px 30px;
 }
 
-@media(max-width: 1024px) {
-    .section-title {
-        margin-top: var( --spacing-05);
-    }
-}
\ No newline at end of file
+@media (max-width: 768px) {
+  .steps_info_box_with_icon {
+    padding-right: 40px;
+  }
+}
+
+@media (max-width: 425px) {
+  .steps_info_box_with_icon {
+    padding: 0 1px 1px;
+  }
+}
+
+.steps_info_box_without_icon {
```

Comment:
Please don't add it here. Add those styles to the react page! :)

.InnerBox .steps_info_box_without_icon

Reply:
much cleaner now thanks